peril-settings.json

```diff
@@ -91,16 +105,24 @@
             ]
         },
         "Automattic/simplenote-macos": {
-            "pull_request, pull_request.labeled, pull_request.unlabeled": [
-                "Automattic/peril-settings@org/pr/ios-macos.ts",
-                "Automattic/peril-settings@org/pr/label.ts",
+            "pull_request": [
+                "Automattic/peril-settings@org/pr/ios-macos.ts"
+            ],
+            "pull_request.opened, pull_request.labeled, pull_request.unlabeled, issues.opened, issues.labeled, issues.unlabeled": [
+                "Automattic/peril-settings@org/pr/label.ts"
+            ],
+            "pull_request.opened, pull_request.synchronize": [
                 "Automattic/peril-settings@org/pr/release-notes.ts"
             ]
         },
         "Automattic/simplenote-android": {
-            "pull_request, pull_request.labeled, pull_request.unlabeled": [
-                "Automattic/peril-settings@org/pr/android.ts",
-                "Automattic/peril-settings@org/pr/label.ts",
+            "pull_request": [
+                "Automattic/peril-settings@org/pr/android.ts"
+            ],
+            "pull_request.opened, pull_request.labeled, pull_request.unlabeled, issues.opened, issues.labeled, issues.unlabeled": [
+                "Automattic/peril-settings@org/pr/label.ts"
+            ],
+            "pull_request.opened, pull_request.synchronize": [
                 "Automattic/peril-settings@org/pr/release-notes.ts"
             ],
```

Comment:
I was surprised to see that `simplenote-android` doesn't run the `milestone.ts` rule, nor the `android-diff-size.ts` rule. Is that intentional?